.gitignore

```diff
@@ -52,3 +52,4 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 /.tmp/
+/.env
```

Comment:
yep, need to update readme.md to explain the whole process (strava app creation + id and secret setting).